tardis/transport/montecarlo/r_packet_transport.py

```diff
@@ -140,7 +140,6 @@ def trace_packet(
         if tau_trace_combined > tau_event and not disable_line_scattering:
             interaction_type = InteractionType.LINE  # Line
             r_packet.last_interaction_in_nu = r_packet.nu
-            r_packet.last_interaction_in_r = r_packet.r
             r_packet.last_line_interaction_in_id = cur_line_id
             r_packet.last_line_interaction_shell_id = r_packet.current_shell_id
```

Comment:
Should I remove line 144, since it might give the incorrect impression that the location is updated here? Although the shell_id doesn't change when we encounter a line interaction.

Reply:
@andrewfullard

Reply:
I agree, it would be best to have all the interaction information update at the same code location as the radius.